spec/models/early_career_payments/eligibility_spec.rb

```diff
@@ -1178,44 +1182,25 @@
     end
   end
 
-  describe "#trainee_teacher_in_2021?" do
-    context "when EarlyCareerPayments policy AcademicYear - 2021" do
-      let(:eligibility) { build_stubbed(:early_career_payments_eligibility, nqt_in_academic_year_after_itt: false) }
+  describe "#trainee_teacher?" do
+    let(:eligibility) { build_stubbed(:early_career_payments_eligibility, nqt_in_academic_year_after_itt: false) }
 
-      before do
-        @ecp_policy_date = PolicyConfiguration.for(EarlyCareerPayments).current_academic_year
-        PolicyConfiguration.for(EarlyCareerPayments).update(current_academic_year: AcademicYear.new(2021))
-      end
-
-      after do
-        PolicyConfiguration.for(EarlyCareerPayments).update(current_academic_year: @ecp_policy_date)
-      end
-
-      it "returns true" do
-        expect(eligibility.trainee_teacher_in_2021?).to be true
-      end
-
-      it "returns false" do
-        eligibility.nqt_in_academic_year_after_itt = true
-        expect(eligibility.trainee_teacher_in_2021?).to be false
-      end
+    before do
+      @ecp_policy_date = PolicyConfiguration.for(EarlyCareerPayments).current_academic_year
+      PolicyConfiguration.for(EarlyCareerPayments).update(current_academic_year: AcademicYear.new(2022))
     end
 
-    context "when EarlyCareerPayments policy AcademicYear - NOT 2021" do
-      before do
-        @ecp_policy_date = PolicyConfiguration.for(EarlyCareerPayments).current_academic_year
-        PolicyConfiguration.for(EarlyCareerPayments).update(current_academic_year: AcademicYear.new(2022))
-      end
-
-      after do
-        PolicyConfiguration.for(EarlyCareerPayments).update(current_academic_year: @ecp_policy_date)
-      end
+    after do
+      PolicyConfiguration.for(EarlyCareerPayments).update(current_academic_year: @ecp_policy_date)
+    end
 
-      it "returns false" do
-        eligibility = EarlyCareerPayments::Eligibility.new(nqt_in_academic_year_after_itt: false)
+    it "returns true" do
+      expect(eligibility.trainee_teacher?).to be true
```

Comment:
Not sure if you like this better, but RSpec magic will let you write: `expect(eligibility).to be_a_trainee_teacher`